modules/user_manual/pages/apps/admin_audit.adoc

```diff
@@ -0,0 +1,198 @@
+= The Admin Audit App
+:toc: right
+:toclevels: 1
+:leveloffset: +1
+
+== Introduction
+
+The Admin Audit app is an auditing module for ownCloud to trace the actions of users and administrators.
+
+== Configuration
+
+Configuration is needed to redirect the messages into a separate file.
+
+Add these lines to "config.php" and adjust the target path accordingly:
```

Comment:
`target` --> `logfile`

Reply:
I've, effectively, done this in the most recent commit.